cmd/kube-controller-manager/app/options/jobcontroller.go

```diff
@@ -32,6 +32,8 @@ func (o *JobControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	if o == nil {
 		return
 	}
+
+	fs.Int32Var(&o.ConcurrentJobSyncs, "concurrent-job-syncs", o.ConcurrentJobSyncs, "The number of job objects that are allowed to sync concurrently. Larger number = more responsive jobs, but more CPU (and network) load")
 }
 
 // ApplyTo fills up JobController config with options.
```

Comment:
@Michael0x2a can you please add validation for both of these controllers like we do for example in stateful sets? https://github.com/kubernetes/kubernetes/blob/57eb5d631ccd615cd161b6da36afc759af004b93/cmd/kube-controller-manager/app/options/statefulsetcontroller.go#L59

Reply:
@atiratree -- done!